cmd/version/version.go

```diff
@@ -78,8 +84,64 @@ func NewCmdVersion(rootCmd *cobra.Command) *cobra.Command {
 			}
 
 			util.TablePrinter(util.VersionColumnDefinition, rows, printers.PrintOptions{Wide: true})
+			getCliLatestVersion(rootCmd.Version)
 		},
 	}
 	cmd.SetUsageTemplate(versionCmdHelp)
 	return cmd
 }
+
+func getCliLatestVersion(currVersion string) {
```

Comment:
I'm sorry for being a grammar nazi here, but the name `getCliLatestVersion` makes me think this function should return the latest version, maybe checking for the latest & current function for differences can happen in another function or maybe this can be renamed as `checkForLatestVersion`.

Reply:
Yeah right, my mistake xD. Changed.